documentation/Building.md

```diff
@@ -66,4 +66,19 @@ docker build -t tiledb/tiledb-vector-search .
 You run the example docker image which provides the python package with:
 ```
 docker run --rm tiledb/tiledb-vector-search
+```
+
+# Formatting
+
+The C++ code is formatted with `clang-format` version 17. Install it yourself or by running:
```

Comment:
Can we get 16? Core is using 16 so that will be easier to keep aligned between projects.

Reply:
So it's simple on linux. But on Mac `brew` doesn't have 16: https://formulae.brew.sh/formula/clang-format - it only has 17:
```
(TileDB-Vector-Search-4) ~/repo/TileDB-Vector-Search-4 brew install clang-format@16
Warning: No available formula with the name "clang-format@16". Did you mean clang-format, clang-format@11 or clang-format@8?
==> Searching for similarly named formulae and casks...
==> Formulae
clang-format                   clang-format@11                clang-format@8

To install clang-format, run:
  brew install clang-format
```
Looking more at Core, I think they actually have a bug, as they do:
```
install_apt_pkg() {
  wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
  add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-16 main' &&
  apt-get update -qq && apt-get install -qq -y clang-format-16
}

install_brew_pkg() {
  brew upgrade && brew install clang-format
}
```
But this will install 16 on linux and 17 on mac. I'm guessing that many people just ran `brew install clang-format` before 17 was out, but anyone new will have 17?

Reply:
I'm ok with (1) then.